modules/linear_boltzmann_solvers/executors/pi_keigen.cc

```diff
@@ -15,42 +15,41 @@ namespace opensn
 namespace lbs
 {
 
-OpenSnRegisterObjectInNamespace(lbs, XXPowerIterationKEigen);
+OpenSnRegisterObjectInNamespace(lbs, PowerIterationKEigen);
 
 InputParameters
-XXPowerIterationKEigen::GetInputParameters()
+PowerIterationKEigen::GetInputParameters()
 {
   InputParameters params = opensn::Solver::GetInputParameters();
 
-  params.SetGeneralDescription("Generalized implementation of a k-Eigenvalue solver using Power "
-                               "Iteration.");
+  params.SetGeneralDescription("Implementation of a k-Eigenvalue solver using Power Iteration");
   params.SetDocGroup("LBSExecutors");
 
-  params.ChangeExistingParamToOptional("name", "XXPowerIterationKEigen");
+  params.ChangeExistingParamToOptional("name", "PowerIterationKEigen");
 
   params.AddRequiredParameter<size_t>("lbs_solver_handle", "Handle to an existing lbs solver");
+
   params.AddOptionalParameter("max_iters", 1000, "Maximum power iterations allowed");
+
   params.AddOptionalParameter("k_tol", 1.0e-10, "Tolerance on the k-eigenvalue");
+
   params.AddOptionalParameter(
-    "reset_solution",
-    true,
-    "Flag, if set to true will initialize the phi-solution to all 1's before "
-    "executing");
+    "reset_solution", true, "If set to true will initialize the phi-solution to 1.0");
```

Comment:
```suggestion
    "reset_phi", true, "If set to true will initialize the flux moments to 1.0");
```
For consistency. See other suggestions.